apis/cloudidentity/v1beta1/group_reference.go

```diff
@@ -75,9 +75,19 @@ func (r *GroupRef) NormalizedExternal(ctx context.Context, reader client.Reader,
 	if err != nil {
 		return "", fmt.Errorf("reading status.externalRef: %w", err)
 	}
-	if actualExternalRef == "" {
+	if actualExternalRef != "" {
+		r.External = actualExternalRef
+	}
+
+	name, _, _ := unstructured.NestedString(u.Object, "status", "name")
```

Comment:
Do we still need to read `status.name` if we already get `actualExternalRef` from `status.externalRef`?

Reply:
Yeah my question was about we should return if `actualExternalRef != ""`. Now that we are returning, this comment is resolved.